app/services/importer/todoist.rb

```diff
@@ -0,0 +1,35 @@
+require 'open-uri'
+require 'json'
+
+class Importer::Todoist
+  API_URL = 'https://todoist.com/API/v7/sync'
+
+  def self.import(token)
+    request_url = create_url(token: token, sync_token: '*', resource_types: '["all"]')
+    todoist_tasks = api_request(request_url)
+    todoist_tasks["items"].map do |todoist_task| 
+      {
+        title: todoist_task["content"],
+        deadline_at: (todoist_task["due_date_utc"].presence || Time.zone.now.since(1.month)),
+        created_at: todoist_task["date_added"],
+        weight: 1
+      }
+    end
+  end
+
+  private
+
+  def self.api_request(request_url)
```

Comment:
__[rubocop]__
* Lint/IneffectiveAccessModifier: `private` (on line 20) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.